src/options.js

```diff
@@ -0,0 +1,44 @@
+// Saves options to chrome.storage
```

Comment:
I was like "wow this options page is written exactly how I would've written it" LMAO

Reply:
Haha thank you chrome documentation